### DISCUSSION TITLE
PKU 编译原理实践 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#609] [Announcements]
# PKU 编译原理实践 - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%BC%96%E8%AF%91%E5%8E%9F%E7%90%86/PKU-Compilers/

**Comment by Unik-lif** [comment]
邢神，超我！

**Comment by winshengly** [comment]
厉害，我是小白，这对我太有用了！！！！

**Comment by leizheng2005** [comment]
走一条人迹罕至的路，走一条可能没有成果的路，来自周围人的压力，来自各种考核积分的压力，没有别人的认可，没有成绩的证明，甚至可能没有未来。
本来想微信发，但不知道发给谁，写了又删掉，好像我们的关系无法说出这些话。
谢谢作者的这句话：
“而这一切，没有学分，没有绩点，没有老师，没有同学，只有一个信念 —— 你在变强。”

**Comment by kirkzhangtech** [comment]
教材和视频课程看哪个呀？

**Comment by Ayn1631** [comment]
没有视频啊啊啊啊啊啊

**Comment by miRceLzeS** [comment]
今天做完，体验很不错，个人感觉nju的编译原理+这门课刚刚好，前者补充编译器前端相关的理论知识(考试面试什么的还是会用得上)，这门正好有适合本科宝宝体质的编译优化内容

**Reply by miRceLzeS** [reply]
另外给和我一样不爱看课的朋友私心推荐我自己的路线 (Lox)[https://www.craftinginterpreters.com/contents.html]的tree-walk interpreter 部分+这门课+ Lox的byte code virtual machine的26-30

**Reply by Der-mark** [reply]
大佬你好，我不是学生了，不能用OJ，有啥办法调试和提交代码去check吗？

**Comment by a12387** [comment]
学时绝对不止60h吧，特别是如果尝试加入优化的情况下

**Comment by akseven6221** [comment]
我去，这助教水平也太高了，设写的的这个koopir感觉很厉害啊，难以想象是自己怎么设计出来的

**Comment by 383494** [comment]
[我的学习历程](https://www.cnblogs.com/x383494/collections/25124)，希望能帮助到后人

**Reply by baoyou10** [reply]
感谢朋友

[a comment by an automated account is not shown]
